File: src/javascript/example/cli/src/commands/color-test.js
import chalk from 'chalk';
import register from '../requests/register.js';
import doCoolStuff from '../requests/do-cool-stuff.js';

const colorTest = async (color, language) => {
  await register(color)
  .then((res) => doCoolStuff(res.body))
  .then((res) => {
    if(!res.doubleCool) {
      throw(chalk.red('OH NO YOUR SHIT BLEW UP AGAIN!!!!'));
    }
    console.log(chalk[color](`Aww yeah! The ${language ? language : ''} server thinks you're ${res.doubleCool}.`));
  })
  .catch(console.error);
};

export default colorTest;

File: src/python/examples/cli/src/commands/color_test.py
from src.requests.register import register
from src.requests.do_cool_stuff import do_cool_stuff

async def color_test(color, language):
    print(f'{color} and {language}')
    res = await register(color, language)
    cool_res = await do_cool_stuff(res)
    coolness = cool_res.json()
    print(f'got cool_res of {coolness}')
